api/src/main/java/org/pmiops/workbench/exceptions/ExceptionUtils.java

```diff
@@ -31,6 +33,19 @@ public static RuntimeException convertGoogleIOException(IOException e) {
     throw new ServerErrorException(e);
   }
 
+  public static RuntimeException convertFirecloudException(ApiException e) {
+    log.log(Level.INFO, "Exception calling FireCloud", e);
```

Comment:
I would probably bump the log level here for the 500-class errors. It's going to hide the cause having Firecloud 500s logged at INFO level (unless this is already being logged somewhere else that I missed).

Reply:
Done.